pkg/koordlet/resmanager/be_reconcile.go

```diff
@@ -38,7 +38,7 @@ func (r *resmanager) reconcileBECgroup() {
 		if extension.GetPodQoSClass(podMeta.Pod) != extension.QoSBE {
 			continue
 		}
-		if getCPUSuppressPolicy(nodeSLO) != v1alpha1.CPUCfsQuotaPolicy {
+		if enable, policy := getCPUSuppressPolicy(nodeSLO); enable && policy != v1alpha1.CPUCfsQuotaPolicy {
```

Comment:
We should check `!enable || policy != CPUCfsQuotaPolicy`.

> `ResourceUsedThresholdWithBE.Enable` is a switch for the *CPUSuppress* feature. And we generally expect the BEReconcile is enabled by default, and if needed to disable it, we use the feature gate BEReconcile. However, CPUSuppress may conflict with BEReconcile when we set SuppressPolicy as `CPUCfsQuotaPolicy` (both modify the cgroups `cpu.cfs_quota_us`), so the policy is checked in BEReconcile.  
> You got part of it right. `CPUCfsQuotaPolicy` does nothing when `ResourceUsedThresholdWithBE.Enable=false`.

Reply:
+1